workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-element-2/client-extension.dev.yaml

```diff
@@ -0,0 +1,5 @@
+assemble:
```

Comment:
blank assemble?

Reply:
yup, that basically skips the assemble step since no files will actually be needed.

Reply:
The holy grail will be when we find a way to skip the whole build step and cut down the overall task time, but it's an optimization and not necessary right now.